spacy_llm/tests/tasks/examples/entity_linking.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
In spaCy, the component is called `entity_linker` so I would use that as name throughout instead of `entity_linking`.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/bfe7dd783ed01cd47847485366f03fb5841cae42.